package.json

```
@@ -5,7 +5,7 @@
   "author": "Alley Interactive",
   "license": "GPL-2.0-or-later",
   "engines": {
-    "node": ">=16.0.0 <21.0.0",
+    "node": ">=16.0.0 <22.0.0",
```

we want to support 22

```suggestion
    "node": ">=16.0.0 <23.0.0",
```

---

Ah yup - thanks for flagging that. Running a bit on empty today 🫤